examples/simple.rs

```diff
@@ -8,10 +8,10 @@ fn main() {
     ctx.add_messages("key1 = Value 1");
     ctx.add_messages("key2 = Value 2");
 
-    let id = String::from("key1");
+    let msg = ctx.get_message("key1").unwrap();
```

Comment:
`unwrap` is usually a sign of a trouble to come. It's useful for mocks, but we should avoid them in code as much as possible.

In this case, since `get_message` returns an Option,  I think there's a value in matching it, and printing an error message in case it returns false (instead of panicking).